cmd/infra/powervs/create.go

```diff
@@ -607,9 +612,8 @@ func getAccount(auth core.Authenticator) (accountID string, err error) {
 }
 
 // createPowerVSSession creates PowerVSSession of type *ibmpisession.IBMPISession
-func createPowerVSSession(powerVSRegion string, powerVSZone string, debug bool) (session *ibmpisession.IBMPISession, err error) {
+func createPowerVSSession(account string, powerVSRegion string, powerVSZone string, debug bool) (session *ibmpisession.IBMPISession, err error) {
 	auth := getIAMAuth()
-	account, err := getAccount(auth)
```

Comment:
also can you add a comment in the PR about this change?

Reply:
Revoked the changes related to account.